### ISSUE TITLE
League of Legends Skin Sale Notifications

### ISSUE THREAD

**Issue by pedrofracassi** [SwitchbladeBot/switchblade#1052]

**Comment by pedrofracassi** [comment]
![image](https://user-images.githubusercontent.com/25179120/67524739-b50c2280-f687-11e9-9a5a-ac6f226c7ccb.png)

We might need to run a League of Legends client somehow and interact with it through the LCU API

**Comment by pedrofracassi** [comment]
https://developer.riotgames.com/docs/lol#league-client

**Comment by giraffesyo** [comment]
Running a LoL client is going to be pretty heavy. If you're going to go that approach why not just scrape the data from someone who is already gathering the information? e.g. https://leagueoflegends.fandom.com/wiki/Sales

Only thing is both of these solutions are fragile.

**Comment by pedrofracassi** [comment]
Yeah, I know of a few bots that scrape the fandom wiki for information. My only concern is that it is manually updated, and any slight difference in the page's formatting might break the bot
